swim/ping_handler.go

```diff
@@ -40,8 +42,11 @@ func handlePing(node *Node, req *ping) (*ping, error) {
 	changes, fullSync :=
 		node.disseminator.IssueAsReceiver(req.Source, req.SourceIncarnation, req.Checksum)
 
+	// Bidirectional full sync.
 	if fullSync {
-		// TODO: handle full sync
+		go func() {
```

Comment:
I think what Jeff means is that if the one way full sync fixes the membership, we don't need the reverse full sync